tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -283,6 +283,7 @@ bool test_day_formatter() {
 
     assert(format(STR("{:%d %d %d}"), day{27}) == STR("27 27 27"));
     assert(format(STR("{:*5%}"), day{200}) == STR("**200"));
+    assert(format(STR("{}"), day{00}) == STR("00 is not a valid day"));

```

Comment:
```suggestion
    assert(format(STR("{}"), day{0}) == STR("00 is not a valid day"));
```

`00` is even more octal, I think this should be plain `0` in code. (We wouldn't write `day{09}`.)